Fix some minor mistakes in LIPs 0040, 0041, and 0065 (#181)
LiskHQ/lips: proposals/lip-0065.md

* `height`: The height of the block in which the event was emitted. Together with the `index`, it uniquely identifies an event within a blockchain.
* `index`: An integer counter (unique within a block) indicating how many events were emitted before the current one.

### Event Tree

Events are emitted during [application domain stages of the block processing][lip-0055#block-processing-stages] and are inserted in a [sparse Merkle tree][lip-0039] as new leaf nodes. The leaf values are set to the serialized events. Leaf keys are given by the concatenation of a generic byte value and an index that identifies the position in which the event was emitted during the block processing. In particular, each event specifies an array of topics (similar to the 'indexed' keyword used in [Solidity](https://docs.soliditylang.org/en/v0.8.11/cheatsheet.html#modifiers)). The same event is inserted multiple times in the sparse Merkle tree, once for each entry in this array. By default, the indexed topics array contains the ID of the transaction that emitted the event or a constant placeholder for events emitted during the processing of the block header.

> However, events emitted during the [processing of a cross-chain message][lip-0053#ccu-processing] contain the ID of the cross-chain message as default topic rather than the ID of the cross-chain update transaction that contained the cross-chain message.

Reviewer: ```suggestion
However, events emitted during the [processing of a cross-chain message][lip-0053#ccu-processing] contain the ID of the cross-chain message as default topic instead of the ID of the cross-chain update transaction that contained the cross-chain message.
```

Reviewer: Not a must, but sounds a bit vague otherwise

#### `module`, `name`, and `data`

The `module` property corresponds to the module to which the event belongs. Each event has a `name` and JSON schema used to serialize and deserialize the `data` property. The `name` is a string of length at most `MAX_EVENT_NAME_LENGTH`. The value `EVENT_NAME_DEFAULT` is reserved for the default event and cannot be assigned.

Both `module` and `name` must be alphanumeric strings (lower and upper case characters allowed).

#### `topics`

The event `topics` is an array of unique byte values whose length ranges between `1` and `MAX_TOPICS_PER_EVENT`. 
By default, the first topic is set to:

* the transaction ID, for events emitted during the transaction processing stages,

> * the cross-chain message ID, for events emitted during the cross-chain message processing stages), 

Reviewer: ```suggestion
* the cross-chain message ID, for events emitted during the cross-chain message processing stages, 
```

* the constant `EVENT_TOPIC_INIT_GENESIS_STATE`, for events emitted during the 'genesis state initialization' stage,

> * `EVENT_TOPIC_FINALIZE_GENESIS_STATE`, for events emitted during the 'genesis state finalization' stage, 

Reviewer: ```suggestion
* the constant `EVENT_TOPIC_FINALIZE_GENESIS_STATE`, for events emitted during the 'genesis state finalization' stage, 
```

> * `EVENT_TOPIC_BEFORE_TRANSACTIONS`, for events emitted during the 'before transactions execution' stage, 

Reviewer: ```suggestion
* the constant `EVENT_TOPIC_BEFORE_TRANSACTIONS`, for events emitted during the 'before transactions execution' stage, 
```
